main.go

```diff
@@ -96,6 +97,7 @@ func main() {
 		os.Exit(1)
 	}
 
+	go internal.InitializeRouter()
```

Comment:
Same comment I threw in the interlal/http.go package: we need to pass a context.Context here instead of building that up under-the-hood.